blocks/default-output-block/output-types/__tests__/default.test.jsx

```diff
@@ -7,6 +7,13 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
+const dummyComp = () => <meta content="dummy" />;
+const mockFuntions = {
+  Libs: dummyComp,
```

Comment:
Not sure I totally understand this mock. can you elaborate?

Reply:
I'm mocking the components that default-output-block use to render the libs, css and fusion data.
https://github.com/WPMedia/fusion-news-theme-blocks/blob/c7da8bccfb8ee711f88274fa1d4f937b42c51340/blocks/default-output-block/output-types/default.jsx#L182

there was warnings on the tests because those components wasn't received.